dist/main.css

```diff
@@ -0,0 +1,94 @@
+.jumbotron {
+  padding-top: 48px;
+  padding-bottom: 48px;
+}
```

Comment:
padding name convertion use standard  [  .padding-v-48{ padding-top: 48px; padding-bottom: 48px;}

Reply:
also for margin do the same it makes things easy

Reply:
Thank you.